cmd/topicctl/subcmd/apply.go

```diff
@@ -315,30 +315,27 @@ func applyTopic(
 		}
 		topicChanges, err := cliRunner.ApplyTopic(ctx, applierConfig)
 		if err != nil {
-			// if one of the steps after updateSettings errors when updating a topic,
-			// we can be in a state where some (but not all) changes were applied
-			// some topic creation errors also still create the topic
+			// If one of the steps after updateSettings errors when updating a topic,
+			// we can be in a state where some (but not all) changes were applied.
+			// Some topic creation errors also still create the topic
 			log.Error("Error detected while creating or updating a topic")
-			if checkForChanges(topicChanges) {
-				log.Error("The following changes were still made:")
-				partialChanges, printErr := printJson(topicChanges)
-				if printErr != nil {
-					log.Error("Error printing JSON changes data")
-				} else {
-					log.Errorf("%#v", partialChanges)
-				}
+			log.Error("The following changes were still made:")
+			partialChanges, printErr := printJson(topicChanges)
+			if printErr != nil {
+				log.Error("Error printing JSON changes data")
+			} else {
+				log.Errorf("%#v", partialChanges)
```

Comment:
this now no longer checks if any changes were made via `checkForChanges()` before printing json if an error occurs. This is because we want to send notifications on errors even if there were no changes made.